include/xtensor/xstridedview.hpp

```diff
@@ -568,6 +571,155 @@ namespace xt
     }
 #endif
 
+    class slice_vector : public std::vector<std::array<long int, 3>>
```

Comment:
Is there a better way than inheriting from std::vector?

I basically want the container functionality (iterators and operator[] and size()), but not the insert stuff.